integrations/syft/src/index.tsx

```diff
@@ -0,0 +1,46 @@
+import {
+    createIntegration,
+    FetchPublishScriptEventCallback,
+    RuntimeContext,
+    RuntimeEnvironment,
+} from '@gitbook/runtime';
+
+import script from './script.raw.js';
+
+type SyftRuntimeContext = RuntimeContext<
+    RuntimeEnvironment<
+        {},
+        {
+            syft_key?: string;
+        }
+    >
+>;
+
+export const handleFetchEvent: FetchPublishScriptEventCallback = async (
+    event,
+    { environment }: SyftRuntimeContext
+) => {
+    const syftId =
+        environment.spaceInstallation.configuration.syft_key ??
```

Comment:
Can you update this to `environment.spaceInstallation?.configuration?.syft_key`

This should fix the error you're seeing in the published site.

Reply:
Oh, cool, will handle it